observer/observer.go

```diff
@@ -12,12 +16,19 @@ import (
 type Observer struct {
 	logger   blog.Logger
 	monitors []*monitor
+	shutdown func()
 }
 
-// Start spins off a goroutine for each monitor and then runs forever.
+// Start spins off a goroutine for each monitor, and waits for a signal to exit
 func (o Observer) Start() {
 	for _, mon := range o.monitors {
 		go mon.start(o.logger)
 	}
-	select {}
+
+	sigChan := make(chan os.Signal, 1)
+	signal.Notify(sigChan, syscall.SIGTERM)
+	signal.Notify(sigChan, syscall.SIGINT)
+	signal.Notify(sigChan, syscall.SIGHUP)
+	<-sigChan
+	o.shutdown()
```

Comment:
If this lands after https://github.com/letsencrypt/boulder/pull/6808, this can become
```suggestion
	defer o.shutdown()
	cmd.WaitForSignal()
```

Reply:
There’s no rush for this, so definitely can just wait on #6808

Reply:
#6808 has landed.